src/servers/exporter.ts

```diff
@@ -189,7 +203,7 @@ const main = async () => {
 
   const port = process.env.PORT || 3000
   logger.info(`Server listening to ${port}, metrics exposed on /metrics endpoint`)
-  await server.listen(port)
```

Comment:
Why did you get rid of ```await``` here?

Reply:
This is not a promised function, so the await is a no-op